include/receivercadmium.hpp

```diff
@@ -0,0 +1,115 @@
+/**
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#ifndef __BOOST_SIMULATION_PDEVS_RECEIVER_HPP__
```

Comment:
There is no problem in this header guard for compiling purpose but as a good practice according to naming rules if you change it to **__RECEIVER_CADMIUM_HPP__** then its better.